android/app/src/main/java/org/fossasia/openevent/core/speaker/SpeakerDetailsActivity.java

```diff
@@ -286,6 +288,21 @@ private void hideEmptyURLButtons() {
         }
 
     }
+
+    private void loadData() {
+        speakerDetailsViewModel.getSpeaker(speakerName).observe(this, speakerData -> {
+            selectedSpeaker = speakerData;
+            loadSpeakerDetails();
+        });
+    }
+
+    @Subscribe
+    public void onBookmarksChanged(BookmarkChangedEvent bookmarkChangedEvent) {
+        Timber.d("Bookmarks Changed");
+        loadData();
```

Comment:
I think this and many other solutions mask a greater problem with the implementation. We can accept these changes. But write a TODO, that this is a workaround and fix needs to be done in the implementation of RealmLiveData

Also, we are moving away from event bus architecture, so we don't want to add new code using the pattern